css/style.css

```diff
@@ -1279,6 +1279,9 @@ body .navbar-light .navbar-toggler-icon {
 .team-description h4 {
   padding: 0;
   font-size: 24px;
-  font-family: 'AvenirRegular';
+  font-family: "AvenirRegular";
 }
 /*# sourceMappingURL=style.css.map */
+#blog-container {
+  width: 90% !important;
```

Comment:
- First of all, don't use `important` if the same task can be done using other ways.
- The width of the cards looks odd
- Make the width & height of the blog cards the same as that of the club ones.
![image](https://user-images.githubusercontent.com/42200276/99180328-4d6dbd00-274b-11eb-9b49-fcd23e019ff4.png)